### PR TITLE
Explicitely describe how to execute tasks at the first deployment of a review app and at each deployment

### PR DESCRIPTION
**Pull request by Soulou**
https://documentation-service-pr409.scalingo.io/platform/app/review-apps

### CODE REVIEW

File: _posts/platform/app/2000-01-01-review-apps.md [hunk 1]
```diff
@@ -40,6 +40,40 @@ As a quick example, here is a sample `scalingo.json` that customize the environm
 
 Here, the value of the environment variable `CANONICAL_HOST_URL` will contain the URL to reach the newly deployed app. The `scalingo.json` configuration always takes precedence on parent app configuration.
 
+## Run a task after the 1st deployment of a review app
+
+When a review app is created, you might want to execute a custom action like seeding its database, this can be done using the `first-deploy` scripts in the manifest `scalingo.json`. If such a property is defined, the given command will be executed as a [postdeploy hook]({% post_url platform/app/2000-01-01-postdeploy-hook %}) after the first deployment.
+
+{% warning %}
+This script replaces the postdeploy hook. If you want to execute both, make sure to adapt the `first-deploy` has the same effect that your `postdeploy` additionally to the special action you want to achieve at the first deployment.
```
Comment (EtienneM) [inline _posts/platform/app/2000-01-01-review-apps.md]: I don't understand the second sentence of this paragraph, starting from `If you want to execute both....`.

File: _posts/platform/app/2000-01-01-review-apps.md [hunk 2]
```diff
@@ -40,6 +40,40 @@ As a quick example, here is a sample `scalingo.json` that customize the environm
 
 Here, the value of the environment variable `CANONICAL_HOST_URL` will contain the URL to reach the newly deployed app. The `scalingo.json` configuration always takes precedence on parent app configuration.
 
+## Run a task after the 1st deployment of a review app
+
+When a review app is created, you might want to execute a custom action like seeding its database, this can be done using the `first-deploy` scripts in the manifest `scalingo.json`. If such a property is defined, the given command will be executed as a [postdeploy hook]({% post_url platform/app/2000-01-01-postdeploy-hook %}) after the first deployment.
```
Comment (EtienneM) [inline _posts/platform/app/2000-01-01-review-apps.md]: "seeding its database, this" :arrow_right: "seeding its database. This"

File: _posts/platform/app/2000-01-01-review-apps.md [hunk 3]
```diff
@@ -40,6 +40,40 @@ As a quick example, here is a sample `scalingo.json` that customize the environm
...
+{
+  "env": {
+    "IS_REVIEW_APP": {
+      "value": "true"
+    }
+  }
+}
+```
+
+And use the environment variable in your postdeploy task to check if its is executed in the scope of a review app or not.
```
Comment (EtienneM) [inline _posts/platform/app/2000-01-01-review-apps.md]: its is :arrow_right: it is

**Comment by Soulou** [comment]
@EtienneM I kind of agree with you, how would you organize the information? `scalingo.json` is for review apps and one-click deploy apps, in both case information is taken from the file.

**Comment by EtienneM** [comment]
I would add a new page just to talk about `scalingo.json`. This page would explain the purpose of this file (configure one-click and review apps), and explain the format of the file. 

On the review app page, we never say explicitly that scalingo.json is used to configure review apps. Maybe after the section "What are review apps?" a section "Configure review apps with scalingo.json" to shortly explain this file and link to the above-mentioned page.

**Comment by Soulou** [comment]
Ok that make sense to me

**Pull request merged**